commit f8b4630ef673486c2c77bd291a08ef132981e149
Author: H.J. Lu <hjl.tools@gmail.com>
Date:   Mon Jul 6 06:38:05 2020 -0700

    x86: Correct bit_cpu_CLFSH [BZ #26208]
    
    bit_cpu_CLFSH should be (1u << 19), not (1u << 20).

diff --git a/sysdeps/x86/cpu-features.h b/sysdeps/x86/cpu-features.h
index 7c46242aad69d427..535b3cb2d25b245b 100644
--- a/sysdeps/x86/cpu-features.h
+++ b/sysdeps/x86/cpu-features.h
@@ -277,7 +277,7 @@ extern const struct cpu_features *__get_cpu_features (void)
 #define bit_cpu_PAT		(1u << 16)
 #define bit_cpu_PSE_36		(1u << 17)
 #define bit_cpu_PSN		(1u << 18)
-#define bit_cpu_CLFSH		(1u << 20)
+#define bit_cpu_CLFSH		(1u << 19)
 #define bit_cpu_DS		(1u << 21)
 #define bit_cpu_ACPI		(1u << 22)
 #define bit_cpu_MMX		(1u << 23)
